lib/client.js

```diff
@@ -254,8 +254,15 @@ var Client = function(options) {
   this._context = options.context || null;
   this.ready = false;
 
-  if (!isOriginValid(this._origin)) {
-    throw new Error('Invalid domain ' + this._origin);
+  if (this._origin && isOriginValid(this._origin)) {
+    var origin_hostname = new URL(this._origin).hostname;
+    this.postMessage('tracker', {
```

Comment:
Wondering if we should make this message a little less guessable, just to avoid accidental calls? e.g. `__track__`